db/migrate/20230713095326_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true, on_delete: :cascade
```

Comment:
- [ ] The foreign key to the user's table should be named `author_id` as you did for the posts table.

Reply:
- Please, in the requirement and on the schema it is only for post table

- [ ] Pay attention that in posts table the column holding the foreign key to the users table should be named author_id